style/vendor/AdminLTE.min.css

```diff
@@ -1,7 +1,5065 @@
-@import url(SourceSansPro/SourceSansPro.css);/*!
+@import url(SourceSansPro/SourceSansPro.css);
```

Comment:
Why did you de-minify this file?

Reply:
That was not supposed to happen. Will minify again.